src/vm/core_test.zig

```diff
@@ -867,6 +870,200 @@ test "deduceOp1 when opcode == .AssertEq, res_logic == .Op1, no op0" {
     try expect(res.?.eq(relocatable.fromU64(7)));
 }
 
+test "deduceOp0 when opcode == .Call" {
+    // ************************************************************
+    // *                 SETUP TEST CONTEXT                       *
+    // ************************************************************
+    var vm = try CairoVM.init(gpa_allocator, .{});
```

Comment:
In test I think this is better to use `std.testing.allocator` allocators which are designed for testing purposes no?

Reply:
We should only use testing allocator yes. As it detects memory leaks and you can customize it’s behavior to reproduce some error cases